src/components-examples/cdk/stepper/cdk-custom-stepper-without-form/example-custom-stepper.css

```diff
@@ -10,24 +11,22 @@
   margin-top: 10px;
 }
 
-.example-active {
-  color: blue;
-}
-
 .example-step {
   background: transparent;
   border: 0;
   margin: 0 10px;
   padding: 10px;
-  color: black;
+  color: inherit;
 }
 
 .example-step.example-active {
-  color: blue;
```

Comment:
Maybe we can pick a color that stands out both on light and dark backgrounds? I believe the point of this example is to make it stand out to illustrate a point, rather than be pretty.

Reply:
Yes right, we discussed that point but there is an issue with maintaining text contrast for both light and dark mode. So instead of using a neutral color, we ended using the text colors.

Reply:
Alright, the `font-weight` that you replaced it with should be fine as well.